### ISSUE TITLE
Unnecessary debug messages shown in console.log

### ISSUE THREAD

**Issue by johackim** [maciex/play-to-kodi#18]
There is some console.log not cool on the official add-on : 

https://addons.mozilla.org/en-US/firefox/addon/play-to-kodi4/
https://addons.mozilla.org/firefox/downloads/file/754119/play_to_kodi-1.9.0-an+fx.xpi

```javascript
// js/content_scripts/context-menu-content-scripts.js
console.log("MSITARZ333333333333!!!!!!!!!");
console.log("MSITARZ222!!!!!!!!!");
console.log("MSITARZ1111!!!!!!!!!");
```

**Comment by undg** [comment]
Very not nice to see it every time im checking console

**Comment by maciex** [comment]
You're absolutely right. That shouldn't be there.

I'm working on a new version with fixes to saving settings (as the Chrome vs Firefox API differs, settings are now lost on Add-on update).
This issue will be fixed in the next version (as providing an official update now would break everyone's saved options).

**Comment by maciex** [comment]
I uploaded a signed beta version (1.9.1beta1) of the add-on.
It should be available in the developer channel of the Firefox Add-ons, but honestly I can't find a way to use it.

> 
Users of your add-on who have chosen to download beta versions will automatically be served the newest beta updates. 
via https://developer.mozilla.org/en-US/Add-ons/Distribution

But I can't find a way to use the beta :| Any ideas?

**Comment by maciex** [comment]
OK. Got it....

Beta versions can be installed from the add-on's web page. Direct link:
https://addons.mozilla.org/en-US/firefox/addon/play-to-kodi4/versions/beta

**Comment by johackim** [comment]
Thanks ;)

**Comment by maciex** [comment]
Fixed in 1.9.1beta1
